ProjectTraning/Task_20_07/ProgramForFridge.cs

```diff
@@ -0,0 +1,78 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProjectTraning.Task_20_07
+{
+    public delegate void FridgeDelegate(object a, FridgeEventArgs e);
+
+    public class ProgramForFridge
+    {
+        public FridgeRefreg Fridge { get; set; }
+
+        public ProgramForFridge()
+        {
+            this.Fridge = new FridgeRefreg();
+
+            this.Fridge.Fridge += EventInvoke;
+
+        }
+
+
+        public void Execute()
+
+        {
+            while (true)
+            {
+
+                Console.WriteLine($"Состояние холодильника: \n{Fridge.StatusOfFridge()}");
+
+                Console.WriteLine("\nВыберите действие \n 1 - включить холодильник \n 2 - выключить холодильник \n 3 - открыть основную дверь \n 4 - закрыть основную дверь \n 5 - открыть морозилку \n 6 - закрыть морозилку \n 7 - выход из программы управления");
+                int result = Convert.ToInt32(Console.ReadLine());
+
+                if(result == 1)
```

Comment:
It is not correct to use 'if' statements for all cases.
Should be used if-else if or switch statement.

Reply:
-1 to source code quality.